StartAKSClusterAssessment.ps1

```diff
@@ -65,17 +92,46 @@ foreach ($currentSubscription in $subscriptions) {
     Write-Host "***** Assessing the subscription $($currentSubscription.displayName) ($($currentSubscription.id)..." -ForegroundColor Cyan
     az account set -s $currentSubscription.SubscriptionId --only-show-errors 
 
-    $jsonAksClusters = az aks list -o json --only-show-errors 
-    $jsonAksClusters | Out-File -FilePath "$OutPath\raw_$today.json" -Append
-    $aksClusters = $jsonAksClusters | ConvertFrom-Json -AsHashTable
-    
-    foreach ($currentAKSCluster in $aksClusters) {
-        Write-Host ""
-        Write-Host "**** Assessing the AKS Cluster $($currentAKSCluster.name)..." -ForegroundColor Blue
-        $aksCluster = [AKSClusterCheck]::new($currentSubscription.id, $currentSubscription.displayName, $currentAKSCluster)
+    # $jsonSQLservers = az sql server list -o json --only-show-errors 
+    # $jsonSQLservers | Out-File -FilePath "$OutPath\raw_$today.json" -Append
+    # $fullservers = $jsonSQLservers | ConvertFrom-Json -AsHashTable
+
+    # Convert JSON strings to PowerShell objects
+$servers = az sql server list -o json --only-show-errors | ConvertFrom-Json
+$firewallRules = az sql server firewall-rule list -o json --only-show-errors | ConvertFrom-Json
 
-        $aksCluster.assess().GetAllResults() | Export-Csv -Path "$OutPath\assess_$today.csv" -NoTypeInformation -Append -Delimiter $csvDelimiter
+# Assuming firewall rules contain a property that can be used to match them to their respective servers,
```

Comment:
lil background, so I needed info from a different code so I combined it in to 1 json and use that as source. Additionally, SQL DB section is included but as comment bcz I'm not sure yet